torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -3020,6 +3020,51 @@ RegisterOperators reg2({
           return 0;
         },
         aliasAnalysisFromSchema()),
+    Operator(
+        "aten::all(int[] self) -> bool",
+        [](Stack& stack) {
+          c10::List<int64_t> l = pop(stack).toIntList();
+          push(stack, std::move(l));
```

Comment:
We should only be pushing onto the stack once, at the end of the function. Here you have pushed onto the stack potentially three times for a function that is only supposed to return one value.

Reply:
Removed the indexing and multiple stack calls in my latest commit. Thanks for the suggestions.